### PR TITLE
Added middy validation example

### PR DESCRIPTION
**Pull request by Manitej66**

### CODE REVIEW

File: _examples/how-to-use-middy-validation-in-your-serverless-app.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+  Status: deployed
+  Outputs:
+    ApiEndpoint: https://51q98mf39e.execute-api.us-east-1.amazonaws.com
+```
+
+The `ApiEndpoint` is the API we just created. Let's test our endpoint using [insomnia](https://insomnia.rest/)
+
+![insomnia_post_request_with_correct_schema](/assets/examples/middy-validation-example/request1.png)
+
+As you can see the endpoint is working absolutely fine. We sent `mani` and `teja` as our `fname` and `lname` respectively and got `Hello, mani-teja` as the response
```
Comment (fwang) [inline _examples/how-to-use-middy-validation-in-your-serverless-app.md:128]: I received `Hello, undefined-undefined.`. `event.body` is a string and `fname` and `lname` cannot be parsed.
Screen Shot 2021-11-10 at 6 03 22 PM

File: _examples/how-to-use-middy-validation-in-your-serverless-app.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+| `@middy/http-json-body-parser` | This middleware automatically parses HTTP requests with a JSON body and converts the body into an object                                                                                                                        |
+| `@middy/http-error-handler`    | Automatically handles uncaught errors that contain the properties `statusCode` (number) and `message` (string) and creates a proper HTTP response for them (using the message and the status code provided by the error object) |
+| `@middy/validator`             | This middleware automatically validates incoming events and outgoing responses against custom schemas defined with the JSON schema syntax.                                                                                      |
+| `ajv`                          | AJV stands for Another JSON Schema Validator and represents the fastest validator for JSON schemas around                                                                                                                       |
+
```
Comment (fwang) [inline _examples/how-to-use-middy-validation-in-your-serverless-app.md:155]: Let's keep the wording for the explanation consistent:
- This middleware parses HTTP requests with a JSON body ...
- This middleware handles uncaught errors that ...
- This middleware validates incoming events and ...

:+1:

**Comment by fwang** [comment]
Hey @Manitej66, in the "Adding response validation" section, let's add a step showing off an invalid response.

**Comment by Manitej66** [comment]
I think it's the same as input schema, so I don't think it's needed. But If
you think it's needed I'll add @Frank Wang ***@***.***>

**Comment by jayair** [comment]
This is perfect @Manitej66!

**Pull request merged**